docs(oss): add light version of streaming docs (#3828)
langchain-ai/docs: src/oss/deepagents/event-streaming.mdx

---

> title: Event Streaming

Reviewer: ```suggestion
title: Event streaming
```

description: Stream subagents, messages, tool calls, and final output from Deep Agents.
---

> Deep Agents build on LangGraph's Event Streaming model and add a first-class `subagents` projection. Use it when you want application-facing streams for the coordinator agent, delegated subagents, nested messages, tool calls, and final state.

Reviewer: ```suggestion
Deep Agents build on LangGraph's event streaming model and add a first-class `subagents` projection. Use it when you want application-facing streams for the coordinator agent, delegated subagents, nested messages, tool calls, and final state.
```

can we crosslink to langgraph docs here?

<Info>

> Want to try Event Streaming in a working project? Start with the [streaming cookbook](https://github.com/langchain-ai/streaming-cookbook) for runnable examples and links to deeper docs.

Reviewer: ```suggestion
Check out the [streaming cookbook](https://github.com/langchain-ai/streaming-cookbook) for runnable examples and links to deeper docs.
```

What does "deeper docs" mean here? Can we also turn this into a `<Tip>`?

Author: Replaced "deeper docs" with "detailed reference documentation"

</Info>

<Note>

> Interested in lower-level Pregel stream modes and the existing Deep Agents streaming guide? See the [Streaming page](/oss/deepagents/streaming).

Reviewer: ```suggestion
For information about lower-level Pregel stream modes, refer to the [Deep Agents streaming](/oss/deepagents/streaming) docs.
```
